<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SchemaEditor</class>
 <widget class="QDialog" name="SchemaEditor">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>395</width>
    <height>273</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>schemaeditor</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>453</y>
     <width>16</width>
     <height>16</height>
    </rect>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::NoButton</set>
   </property>
  </widget>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>9</x>
     <y>9</y>
     <width>378</width>
     <height>256</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QListWidget" name="fieldsListWidget"/>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="addFieldButton">
        <property name="text">
         <string>Додати поле</string>
        </property>
        <property name="icon">
         <iconset resource="../../resources.qrc">
          <normaloff>:/icons/plus-circle.svg</normaloff>:/icons/plus-circle.svg</iconset>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLineEdit" name="schemaNameLineEdit"/>
      </item>
      <item>
       <widget class="QLineEdit" name="fieldNameLineEdit"/>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QDialogButtonBox" name="buttonBox_2">
      <property name="standardButtons">
       <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources>
  <include location="../../resources.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>SchemaEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>243</x>
     <y>433</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>453</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>SchemaEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>319</x>
     <y>435</y>
    </hint>
    <hint type="destinationlabel">
     <x>311</x>
     <y>458</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_2</sender>
   <signal>accepted()</signal>
   <receiver>SchemaEditor</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>197</x>
     <y>136</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_2</sender>
   <signal>rejected()</signal>
   <receiver>SchemaEditor</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>197</x>
     <y>251</y>
    </hint>
    <hint type="destinationlabel">
     <x>309</x>
     <y>270</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
